src/test/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsDetailsMappingTest.java

```diff
@@ -443,7 +521,8 @@ void testGetFacilitiesRequired() {
         + "getListingComments returns all the comments separated by newlines")
     @ParameterizedTest
     @CsvSource(value = {
-        "AppellantComments,'OpComments1',Appellant - Mx Test Appellant:\\nAppellantComments\\n\\nOther Party - Mx Test OtherParty:\\nOpComments1",
+        "AppellantComments,'OpComments1',Appellant - Mx Test Appellant:\\nAppellantComments\\n\\nOther Party - "
+            + "Mx Test OtherParty:\\nOpComments1",
```

Comment:
minor, line break at the \\n\\n, makes more sense

Reply:
Done